src/pages/signin/ChangeExpensifyLoginLink.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import PressableWithFeedback from '@components/Pressable/PressableWithFeedback';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Credentials} from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+type ChangeExpensifyLoginLinkOnyxProps = {
+    /** The credentials of the person logging in */
+    credentials: OnyxEntry<Credentials>;
+};
+
+type ChangeExpensifyLoginLinkProps = ChangeExpensifyLoginLinkOnyxProps & {
+    onPress: () => void;
+};
+
+function ChangeExpensifyLoginLink({credentials, onPress}: ChangeExpensifyLoginLinkProps) {
+    const styles = useThemeStyles();
+    const {translate, formatPhoneNumber} = useLocalize();
+
+    return (
+        <View style={[styles.changeExpensifyLoginLinkContainer, styles.mt3]}>
+            {!isEmptyObject(credentials?.login) && <Text style={styles.mr1}>{translate('loginForm.notYou', {user: credentials?.login ? formatPhoneNumber(credentials.login) : ''})}</Text>}
```

Comment:
You can try to update formatPhoneNumber typing (I [did it](https://github.com/Expensify/App/pull/34718/commits/fc2ab2b09ec50eb5b15c9f150d53bc54da86aba6) in one of my PRs as well)
```suggestion
            {!isEmptyObject(credentials?.login) && <Text style={styles.mr1}>{translate('loginForm.notYou', {user: formatPhoneNumber(credentials.login)})}</Text>}
```